### PR TITLE
Data: Imshk

### PR DESCRIPTION
**Pull request by Imshk**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4658"></a>

### CODE REVIEW

**Review by accodes21 — APPROVED** [review]

**Review by himanshu1221 — DISMISSED** [review]
**LGTM :+1:
Thanks for contributing:tada:**

**Review by Hrishi5111998 — CHANGES_REQUESTED** [review]
hey @Imshk  your json file should be in data folder. please put it in data folder

File: Imshk.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "url": "https://twitter.com/Shashankoffice",
+      "icon": "FaTwitter"
+  },
+    {"name": "Follow me on Github ",
+      "url": "https://github.com/Imshk",
+      "icon": "FaGithub"
+    },
+    {"name": "Follow me on linkedIn ",
+      "url": "https://www.linkedin.com/in/shashankoffice/",
+      "icon": "FaGithub"
```
Comment (Hrishi5111998) [inline Imshk.json:17]: icon value should be **FaLinkedin** . please update!

Comment (Imshk) [reply Imshk.json]: Thank you @Hrishi5111998 , I have updated .

File: Imshk.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "url": "https://twitter.com/Shashankoffice",
+      "icon": "FaTwitter"
+  },
+    {"name": "Follow me on Github ",
+      "url": "https://github.com/Imshk",
+      "icon": "FaGithub"
+    },
+    {"name": "Follow me on linkedIn ",
+      "url": "https://www.linkedin.com/in/shashankoffice/",
+      "icon": "FalinkedIn"
```
Comment (kumarsonsoff3) [inline Imshk.json:17]: You're probably using an `invalid` icon here. It should be `FaLinkedin`.
```suggestion
      "icon": "FaLinkedin"
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Also, move your file to `data` directory. To do so, you need to rename your [file here](https://github.com/Imshk/LinkFree/edit/Imshk-patch-1/Imshk.json) to `data/Imshk.json`

Change `Imshk.json`  - >  `data/Imshk.json` here. 

Let us know if you had any confusion. :)
image

**Comment by Imshk** [comment]
> Also, move your file to `data` directory. To do so, you need to rename your [file here](https://github.com/Imshk/LinkFree/edit/Imshk-patch-1/Imshk.json) to `data/Imshk.json`
> 
> Change `Imshk.json` - > `data/Imshk.json` here.
> 
> Let us know if you had any confusion. :) image

Thank you for helping out with the file here link, I have changed . hope everything is fine now .

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by Imshk** [comment]
please have a review on my PR and if there is still any problem ......mention please, So, I can fix it and get my PR merged soon .

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉 

<p align="center" /></p>

**Comment by Imshk** [comment]
Thank you @kumarsonsoff3 for approving my request and helping me out with things .

**Pull request merged**